### PR TITLE
Manage pagination reset

### PR DESCRIPTION
**Pull request by MirekSobota**
Changes added:

- Implemented an action that resets the page number when switching to a different path.
- Added a pointer cursor to the pagination component

### CODE REVIEW

File: src/common/Pagination/styled.js [hunk 1]
```diff
@@ -28,7 +30,8 @@ export const Button = styled.button`
   border-radius: 5px;
   padding: 8px 16px;
   background-color: ${({ theme }) => theme.color.pattensBlue};
-
+  cursor: pointer;
```
Comment (PawelOwiesek) [inline src/common/Pagination/styled.js:33]: 👌

**Review by PawelOwiesek — APPROVED** [review]
Everything works good , I didn't found any problems. Good job.

File: src/common/Pagination/styled.js
Comment (DorotaZielinska) [inline src/common/Pagination/styled.js:1]: Well done :)

**Review by DorotaZielinska — APPROVED** [review]

File: src/common/Pagination/styled.js
Comment (MirekSobota) [reply src/common/Pagination/styled.js:1]: Thanks!

**Pull request merged**